packages/playground/wordpress/src/boot.ts

```diff
@@ -201,7 +203,11 @@ export async function bootWordPress(
 		);
 	}
 
-	if (options.wordPressZip && !options.dataSqlPath) {
+	const shouldInstall =
+		(options.installWordPress ?? Boolean(options.wordPressZip)) &&
+		!options.dataSqlPath;
```

Comment:
I don't like the naming and the semantics here. If we override, then having `dataSqlPath` should not affect it further. Perhaps this would work better?

```suggestion
	const shouldInstall =
		options.installWordPress ??
		(Boolean(options.wordPressZip) && !options.dataSqlPath);
```